train.py

```diff
@@ -64,6 +64,11 @@
   flags.DEFINE_integer(
       "batch_size", 1024,
       "How many examples to process per batch for training.")
+  
+  flags.DEFINE_integer(
+    "epochs", 100,
```

Comment:
I'd prefer to use "num_epochs" for the name.
Regarding the default number, I would actually be inclined to go in the opposite direction and set it lower, to 5 or 10. This will limit the amount of credit that gets consumed by runaway jobs. Many people are expecting the training to stop by itself.